tests/yam/agama/boot_full_disk_encryption.pm

```diff
@@ -15,7 +15,8 @@ use testapi;
 sub run {
     my $enter_passphrase_for_root = $testapi::distri->get_enter_passphrase_for_root();
     my $enter_passphrase_for_swap = $testapi::distri->get_enter_passphrase_for_swap();
-    my $grub_menu = $testapi::distri->get_grub_menu_installed_system();
+    #my $grub_menu = $testapi::distri->get_grub_menu_installed_system();
+    my $grub_menu = (get_var('FLAVOR') =~ /Leap/) ? $testapi::distri->get_grub_menu_leap_installed_system() : $testapi::distri->get_grub_menu_tombleweed_installed_system();
```

Comment:
please ask in our channel how to try to resolve this kind of situations before be tempted to use conditional code, this is whole point to use Page Object, composition and Distri pattern, to avoid to include `if/else`s.
You will not need this once you add the change in the correct file.